internal/provider/backup/backup_schema.go

```diff
@@ -0,0 +1,137 @@
+					Required:            true,
+				},
+				Resource: &schemaR.StringAttribute{
+					Validators: []validator.String{
+						stringvalidator.OneOf("vdc", "vapp", "vm"),
+					},
+					PlanModifiers: []planmodifier.String{
+						stringplanmodifier.RequiresReplace(),
+					},
+				},
+			},
+			"target_id": superschema.SuperStringAttribute{
+				Common: &schemaR.StringAttribute{
+					MarkdownDescription: "The ID of the target. A target can be a VDC, a VApp or a VM.",
+					Optional:            true,
+				},
+				Resource: &schemaR.StringAttribute{
+					Validators: []validator.String{
+						stringvalidator.ExactlyOneOf(path.MatchRoot("target_id"), path.MatchRoot("target_name")),
+						fstringvalidator.IsUUID(),
+					},
+					PlanModifiers: []planmodifier.String{
+						stringplanmodifier.RequiresReplaceIfConfigured(),
+						stringplanmodifier.UseStateForUnknown(),
+					},
+				},
+				DataSource: &schemaD.StringAttribute{
+					Computed: true,
+				},
+			},
+			"target_name": superschema.SuperStringAttribute{
+				Common: &schemaR.StringAttribute{
+					MarkdownDescription: "The name of the target. A target can be a VDC, a VApp or a VM.",
+					Optional:            true,
+				},
+				Resource: &schemaR.StringAttribute{
+					Validators: []validator.String{
+						stringvalidator.ExactlyOneOf(path.MatchRoot("target_id"), path.MatchRoot("target_name")),
+					},
+					PlanModifiers: []planmodifier.String{
+						stringplanmodifier.RequiresReplaceIfConfigured(),
+						stringplanmodifier.UseStateForUnknown(),
+					},
+				},
+				DataSource: &schemaD.StringAttribute{
+					Computed: true,
+				},
+			},
+			"policies": superschema.SuperSetNestedAttribute{
+				Common: &schemaR.SetNestedAttribute{
+					MarkdownDescription: "The backup policies of the target.",
+				},
+				Resource: &schemaR.SetNestedAttribute{
+					Required: true,
+					Validators: []validator.Set{
+						setvalidator.SizeAtLeast(1),
+					},
+				},
+				DataSource: &schemaD.SetNestedAttribute{
+					Computed: true,
+				},
+				Attributes: map[string]superschema.Attribute{
+					"policy_id": superschema.SuperInt64Attribute{
+						Common: &schemaR.Int64Attribute{
+							MarkdownDescription: "The ID of the backup policy.",
+							Computed:            true,
+						},
+					},
+					"policy_name": superschema.SuperStringAttribute{
+						Common: &schemaR.StringAttribute{
+							MarkdownDescription: "The name of the backup policy.",
+						},
+						Resource: &schemaR.StringAttribute{
+							Required: true,
+							Validators: []validator.String{
+								stringvalidator.OneOf("D6", "D30", "D30NQ", "D60", "W4", "M3", "M12", "XD6", "XD30", "XD60", "XW4", "XM3", "XM12"),
+							},
+							MarkdownDescription: "Each letter represent a strategy predefined: D = Daily, W = Weekly, M = Monthly, X = Replication, The number is the retention period. Please refer to the documentation for more information. https://wiki.cloudavenue.orange-business.com/wiki/Backup",
```

Comment:
Move this to common markdown description and change link to markdown format https://www.markdownguide.org/basic-syntax/#links

Reply:
Fix